### ISSUE TITLE
db append byte (not a feature)

### ISSUE THREAD

**Issue by froggestspirit** [rednex/rgbds#208]
db -1, 0, 1,
compiles as
FF 00 01 00
If you leave the comma at the end.
This did not throw an error when compiling.

**Comment by AntonioND** [comment]
This happens with 16-bit values as well (with `DW`), and 32-bit values (with `DL`).

This one is tricky, I actually think this is the intended behaviour:
https://github.com/rednex/rgbds/blob/8954858bf79a9ad48eacb063d42b47882ede1175/src/asm/asmy.y#L989
https://github.com/rednex/rgbds/blob/8954858bf79a9ad48eacb063d42b47882ede1175/src/asm/asmy.y#L998
https://github.com/rednex/rgbds/blob/8954858bf79a9ad48eacb063d42b47882ede1175/src/asm/asmy.y#L1007

In short, that means that if an entry is empty, it should be considered 0 (`out_Skip()`). You can even put two commas together and they count as an empty entry: `DB 1,,2` generates `$01, $00, $02`.

I'm a bit concerned that someone may be using this and the code could break if I turn this into an error. I could add a warning, though.

**Comment by froggestspirit** [comment]
A warning would probably be good for people that unintentionally do that

**Comment by AntonioND** [comment]
Actually, it is in the documentation. I'll add the warnings (and fix the keyword `DL`, which is broken and undocumented).

https://github.com/rednex/rgbds/blob/8954858bf79a9ad48eacb063d42b47882ede1175/src/asm/rgbasm.5#L571-L580

**Comment by AntonioND** [comment]
Fixed in https://github.com/rednex/rgbds/pull/209
